src/init.jsx

```diff
@@ -1,36 +1,38 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import { createStore, combineReducers } from 'redux';
+import { applyMiddleware, createStore, combineReducers } from 'redux';
 import { Provider } from 'react-redux';
+import createLogger from 'redux-logger';
 
 import Application from './components/Application/Application.jsx';
 
 // Reducers
 import chat from './store/Chat/reducer';
 
+
+// middleware
+const logger = createLogger();        // TODO: make DEV only.
```

Comment:
Note that, prior to pushing to production we'll customize webpack to allow env variables in the client ... I think @broneks mentioned that idea earlier... so we can have conditional middleware for dev / prod.